Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -123,7 +123,7 @@ def __init__(self):
         self.preprocessors = None
         self.outdated_settings = False
         self.setup_layout()
-        QTimer.singleShot(0, self.apply)
+        QTimer.singleShot(0, self.update_learner)
```

Comment:
I see. Still, I think it's too dangerous to skip the potentially overridden `apply`. It can lead to a situation in which the widget will, for instance, ignore some setting when force-applied.

Let's require all widgets derived from `OWBaseLearner` to have `unconditional_apply`. All that don't, need to be fixed. Linear regression, in particular, doesn't need to define `commit` at all (partially my fault :()